aries-site/src/pages/components/skeleton.mdx

```diff
@@ -16,15 +16,12 @@ import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
 </Example>
 
 ### When to use
-Skeletons are used on all pages to provide a visual layout of the content being loaded.  They help users perceive a loading period as being shorter in duration.
+- The interface is content-rich, with numerous elements such as images, text, and other media that take time to load and are known.
```

Comment:
It feels like we could consolidate these 2 bullets into one concise one:

- When data has not yet loaded use Skeleton to provide a scaffold for the page to allow the user have a sense of the page structure and to minimize layout shifts when content is loaded in.

Reply:
I think using the "content-rich" expression might help the users understand a bit better the difference between skeleton and spinner. So, how about this: 
When to use:
- The interface is content-rich and has asynchronous loading, to stabilize layout and prevent shifts during content retrieval.

Reply:
A happy medium?

- When the interface is content-rich and has asynchronous loading, use Skeleton to provide the user with a sense of the page structure and to minimize layout shifts when content is loaded in.